### PR TITLE
Feat : Added signup and login page

### PR DESCRIPTION
**Pull request by sushil-mod**
Added:
1. login and signup (html and css)
2. navigation ( from CL)
login-e
signup
YOU CAN VIEW LOGIN PAGE BY CLICKING LOGIN BUTTON ON HEADER

### CODE REVIEW

**Comment by psychoactiv** [comment]
You can merge

**Review by psychoactiv — APPROVED** [review]

File: screens/css/login.css [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    color: var(--text-color);
+    width: 30rem;
+}
+
+.form-logo{
+
+    border-bottom: 2px solid var(--text-color);
+    
+}
+
```
Comment (fahd23) [inline screens/css/login.css:20]: You can avoid some unnecessary spaces

**Review by fahd23 — APPROVED** [review]
Given one sugeestions, Everything else Looks good to merge.

**Review by atharvabhanage02 — APPROVED** [review]
You can have the cursor pointer property on both Login and SignUp  Button . 
Else everything looks fine and in place

File: screens/css/login.css (hunk 1 above)
Comment (sushil-mod) [reply screens/css/login.css:20]: ohk i will do it , thanks

**Pull request merged**